client/me/purchases/manage-purchase/notices.tsx

```diff
@@ -103,6 +103,15 @@ class PurchaseNotice extends Component<
 		if ( isMonthly( purchase.productSlug ) ) {
 			const daysToExpiry = moment( expiry.diff( moment() ) ).format( 'D' );
 
+			if ( isTemporarySitePurchase( purchase ) ) {
+				return translate( '%(purchaseName)s will expire and be removed %(expiry)s days. ', {
+					args: {
+						purchaseName: getName( purchase ),
+						expiry: daysToExpiry,
+					},
+				} );
+			}
+
 			return translate(
 				'%(purchaseName)s will expire and be removed from your site %(expiry)s days. ',
```

Comment:
I know this is an existing string and not part of this diff, but it seems to suffer from the same issue I mentioned above. 

"Product will expire and be removed 5 days" does not make much sense.